test/testenv/util.go

```diff
@@ -521,3 +521,15 @@ func GetConfLineFromPod(podName string, filePath string, ns string, configName s
 	}
 	return config, err
 }
+
+// ExecuteCommandOnPod execute command on given pod and return result
+func ExecuteCommandOnPod(deployment *Deployment, podName string, stdin string) (string, error) {
```

Comment:
Can't we use PodExecCommand directly? Any reason why we are creating a function?

Reply:
plan to use the ExecPodCommand for multiple verification. So added a function for duplicate code